.github/workflows/kudu.yml

```
@@ -21,7 +29,9 @@ jobs:
     outputs:
       codechange: ${{ steps.filter.outputs.codechange }}
     steps:
-    # For pull requests it's not necessary to checkout the code
+    - uses: "actions/checkout@v4"
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)